python/lsst/afw/image/exposure/_exposureContinued.py

```diff
@@ -94,6 +96,25 @@ def setVariance(self, variance):
 
     variance = property(getVariance, setVariance)
 
+    def getConvexPolygon(self):
+        """Get the convex polygon associated with the bounding box corners.
+
+        Returns
+        -------
+        convexPolygon : `lsst.sphgeom.ConvexPolygon`
+            Returns `None` if exposure does not have a valid WCS.
```

Comment:
Should this be a `raise` instead?

Reply:
I first wrote this with a raise, but then I thought about it some more and decided that if it's okay that `wcs` is `None` then it would make sense that `convex_polygon` would also be `None`.  Though I could change it, I see both ways, but as a property I think it's better if it doesn't raise.